### PR TITLE
Address GCC 9.1 ABI change notice.

### PR DESCRIPTION
**Pull request by hidmic**
By forcing bit-field alignment.

### CODE REVIEW

File: include/mimick/mock.h [hunk 1]
```diff
@@ -46,7 +46,10 @@ struct mmk_params {
     struct mmk_params *next;
 };
 
-struct mmk_mock_options {
+// Force bit-field alignment to circumvent a bug in GCC
+// versions 6, 7, and 8, and a fixup notice on GCC 9.
+// See https://gcc.gnu.org/bugzilla/show_bug.cgi?id=88469.
+struct __attribute__((aligned(32))) mmk_mock_options {
```
Comment (clalancette) [inline include/mimick/mock.h:52]: From https://gcc.gnu.org/onlinedocs/gcc-9.3.0/gcc/Common-Type-Attributes.html#Common-Type-Attributes, my understanding is that the `aligned` attribute is in bytes.  Do you really mean to align it to 32-bytes?

Also, is this going to work when compiling with clang/MSVC?

Comment (hidmic) [reply include/mimick/mock.h:52]: > Do you really mean to align it to 32-bytes?

I meant 4 bytes... :man_facepalming: 

> Also, is this going to work when compiling with clang/MSVC?

Nope... :man_facepalming: 

That's what happens when you do things in a rush and half-burned.

Comment (hidmic) [reply include/mimick/mock.h]: See https://github.com/ros2/Mimick/pull/10/commits/a95e17499a595bb8400162bc03728f4ab534e5c8

**Comment by hidmic** [comment]
@clalancette since this repo isn't part of our workspace, there's no easy way to run CI. We will have a chance when we update the tag in `ros2/mimick_vendor`. I'll tag you there.

**Pull request merged**